src/API/Unsupported/ChannelEvent.ts

```diff
@@ -0,0 +1,54 @@
+import Channel, { ChannelData } from '../Channel/';
+import Twitch from '../../';
+import { NonEnumerable } from '../../Toolkit/Decorators';
+
+export interface ChannelEventData {
+	_id: string;
+	start_time: string;
+	end_time: string;
+	time_zone_id: string;
+	title: string;
+	description: string;
+	cover_image_url: string;
+	language: string;
+	channel: ChannelData;
+	// game: GameData;
+}
+
+export default class ChannelEvent {
+	@NonEnumerable _client: Twitch;
+
+	constructor(private _data: ChannelEventData, client: Twitch) {
+		this._client = client;
+	}
+
+	get channel() {
+		return new Channel(this._data.channel, this._client);
+	}
+
+	get id() {
+		return this._data._id;
+	}
+
+	get startTime() {
+		return Date.parse(this._data.start_time);
+	}
+
+	get endTime() {
+		return Date.parse(this._data.end_time);
+	}
+
+	get title() {
+		return this._data.title;
+	}
+
+	get description() {
+		return this._data.description;
+	}
+
+	coverImageUrl(width: number, height: number) {
```

Comment:
Methods that are not getters should include a meaningful verb prefix. I think "build" would make sense here.